LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2023 The Gorilla Authors. All rights reserved.
+Copyright (c) 2025 The Gorilla Authors. All rights reserved.
```

Comment:
This looks incorrect; this should either contain the year of  the first commit, a range of years, or no year at all (copyright is implied based on date of publication of  each change, which can be found from source control)